vm/src/stdlib/os.rs

```diff
@@ -2777,25 +2783,32 @@ mod nt {
 
     pub const SYMLINK_DIR_FD: bool = false;
 
-    #[pyfunction]
-    pub(super) fn symlink(
+    #[derive(FromArgs)]
+    pub(super) struct SimlinkArgs {
+        #[pyarg(any)]
         src: PyPathLike,
+        #[pyarg(any)]
         dst: PyPathLike,
+        #[pyarg(flatten)]
         target_is_directory: TargetIsDirectory,
-        _dir_fd: DirFd<0>,
-        vm: &VirtualMachine,
-    ) -> PyResult<()> {
+        #[pyarg(flatten)]
+        dir_fd: DirFd<{ SYMLINK_DIR_FD as usize }>,
```

Comment:
Thanks! That's true. Then, `_dir_fd` here looks also safe. Here is an unused field warning.

Reply:
@youknowone Done!